scan/report-en.tex

```diff
@@ -69,7 +69,12 @@
 \item [Participant Date of Birth] \textbf{\VAR{birth_date|e}}
 \item [Specimen Identifier] \textbf{\VAR{qrcode|e}}
 \item [Date Sample Submitted] \textbf{\VAR{collect_ts|e}}
-\item [Date Results Provided] \textbf{\VAR{result_ts|e}}
+\item [Date Results Provided]
+  %- if status_code == "never-tested"
+  \textbf{Never tested}\\
```

Comment:
The trailing `\\` results in an extra line break:

![image](https://user-images.githubusercontent.com/79913/78166601-e5f43e80-7401-11ea-9230-2451532387b4.png)

Would you remove it?